src/integration-tests/commonTests.js

```diff
@@ -122,11 +122,11 @@ const verifyUserRole = async () => {
 			process.env.CLOUD_ENDPOINT
 		)
 
-		console.log('Calling verifyUserRole...')
-		const result = await verifyUserRole()
-		console.log('verifyUserRole result:', result)
+		// console.log('Calling verifyUserRole...')
+		// const result = await verifyUserRole()
+		// console.log('verifyUserRole result:', result)
```

Comment:
can we remove unwanted comments? @priyanka-TL

Reply:
Yes I will do